.github/workflows/conda.yml

```diff
@@ -51,10 +54,7 @@ jobs:
                            --no-anaconda-upload \
                            --output-folder .
       - name: Upload conda package
-        if: |
-          github.event_name == 'push'
-          && github.ref == 'refs/heads/main'
-          && github.repository == 'dask/dask'
+        if: github.event_name == 'push' && github.repository == 'dask/dask'
```

Comment:
Does this condition mean we don't push a new package on tags?

Reply:
Checking the docs for `push` events, it looks like they cover both pushed tags and commits:

https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#push

The main change here is removing the check on `github.ref`, which would've made it so packages would only get updated on pushed _commits_, and essentially is just a repetition of the `on.push` condition we have set:

https://github.com/dask/dask/blob/b663dca0fa4ca4686b8c08f7cb30d11320012901/.github/workflows/conda.yml#L2-L5